vogon/heroku_settings.py

```diff
@@ -178,3 +178,7 @@
         'LOCATION': 'default_cache_table',
     }
 }
+
+# SuperUser Credentials
+USER_ID = 'test'
```

Comment:
These shouldn't be hardcoded -- see the documentation that I sent you earlier about the settings module. These values should be set in environment variables, and then loaded here via ``os.environ``. Also, please use informative variable names (``USER_ID`` for what?).